### DISCUSSION TITLE
First thoughts about Fladder

### DISCUSSION THREAD

**Discussion by alexq83** [DonutWare/Fladder#120] [General]
First of all thanks for this awesome app. I came across this through the unRAID community applications and I really like the media centered approach you've chosen, but I also see a couple of issues here and there. A couple of things I noticed, and you might already be aware of, in no particular order:

- Subtitles: Fladder always selects subtitles if they are present even though they do not fit the settings on the server, e.g. I have german subtitles (CC) but I have set jellyfin to only display forced subtitles. This works for the webclient and the jellyfin apps but not for Fladder (and also not for Streamyfin) 
- Episode screenshots as banners: especially for older shows the episode screenshots don't have the resolution to be displayed full-screen (in landscape) or zoomed in (in portrait). Wouldn't it be better to use the whole screenshot (in portrait) and downsize the screenshot to about a third/quarter of the screen and use this space to display the playback controls? Plex and Emby do something similar, even though I think their approaches could also use work. 

Have you considered creating a discord server? I think this could attract more people to engage in a discussion about Fladder.

**Comment by PartyDonut** [comment]
First of all thank you 😃 

1. The subtitles on Jellyfin have always been a painful point for me. At the moment the Jellyfin settings are fetched and those options are selected initially. I think however that there might be an additional check that the clients themselves do, in the case of the force subtitles option. Not sure how to proceed with this we might have to take a look at how the jellyfin official web/clients do this.
2.  Do you happen to have any screenshots for this? I'm pretty sure I know what you mean but just in case a screenshot would be helpful to make sure.

I have thought about opening a Discord server. However it does require some upkeep and I'm not sure if thats something I would like to do at the moment.

**Comment by alexq83** [comment]
1. Yes, I know that subtitles are a difficult topic as nearly every client I tried had problems with them. Findroid is the only client I know that displays the correct subtitles, except from the official clients.

2. I will get you some screenshots of what I mean. I have to admit though that I'm primarily seeing this with a TV version in mind, as I primarily use a TV to consume media. Here are 2 screenshots showing emby and Fladder in the episode view in portrait. 

<img src="https://github.com/user-attachments/assets/d4842f8d-bd1c-49cf-8edc-582e39416e8e" width=45% height=45%>
<img src="https://github.com/user-attachments/assets/7bc7272f-f003-4185-b266-db5187ce8026" width=45% height=45%>

**Comment by PartyDonut** [comment]
Thanks for the info! 
I've created 2 tickets for the subtitles and the episode banner #128 #127 .

**Comment by bazzadazza72** [comment]
> I have thought about opening a Discord server. However it does require some upkeep and I'm not sure if thats something I would like to do at the moment.

I'd be happy to help build/moderate a Discord server, but if you're not interested yet, I understand.
